### PR TITLE
Rt 1.26 git issue fix

### PR DESCRIPTION
**Pull request by mastarkey**
Added deviations for Cisco unsupported scenarios.

### CODE REVIEW

File: internal/deviations/deviations.go [hunk 1]
```diff
@@ -935,4 +935,25 @@ func SkipBgpSendCommunityType(dut *ondatra.DUTDevice) bool {
...
 	return lookupDUTDeviations(dut).GetBgpActionsSetCommunityMethodUnsupported()
+
+}
+
+// IPv4StaticRouteWithIPv6NextHopUnsupported unsupported ipv4 with ipv6 nexthop
+func IPv4StaticRouteWithIPv6NextHopUnsupported(dut *ondatra.DUTDevice) bool {
+	return lookupDUTDeviations(dut).GetIpv4StaticRouteWithIpv6NhUnsupported()
+}
+
+// IPv6StaticRouteWithIPv4NextHopUnsupported unsported ipv4 with ipv4 next-hop
```
Comment (nandanarista) [inline internal/deviations/deviations.go:946]: typos: 
unsupported
ipv_6_ with ipv4 next-hop

Comment (mastarkey) [reply internal/deviations/deviations.go]: i belive your talking about this line `unsported ipv4 with ipv4 next-hop` this is changed to `unsported ipv6 with ipv4 nexthop`

Comment (nandanarista) [reply internal/deviations/deviations.go]: yes

**Comment by mastarkey** [comment]
@nandanarista / @thesrinath can we please merge this if there are no further comments

**Comment by mastarkey** [comment]
@nandanarista / @thesrinath can you please merge this if there are no further request here ?

File: proto/metadata.proto [hunk 2]
```diff
@@ -491,6 +491,15 @@ message Metadata {
     bool set_no_peer_group = 175;
     // Bgp community member is a string
     bool bgp_community_member_is_a_string = 176;
+    // support for ipv4 to ipv6 next hop
+    bool ipv4_static_route_with_ipv6_nh_unsupported = 177;
+    // support for ipv6 to ipv4 next hop
+    bool ipv6_static_route_with_ipv4_nh_unsupported = 178;
+    // support for static route with drop nh
+    bool static_route_with_drop_nh = 179;
+    // support for static route with drop nh
```
Comment (nandanarista) [inline proto/metadata.proto:500]: this comment isn't correct, (copy pasted from the previous one)

otherwise lgtm

**Review by nandanarista — APPROVED** [review]

**Review by ram-mac — COMMENTED** [review]
LGTM; The tests are successfully run on Cisco devices and all tests passed.
Just one ssuggestion, the gnmi.Await() timer under function ("func (td *testData) testStaticRouteECMP(t *testing.T)") -> Line no 434, 435 needs to be increased for consistent results. Reference below.

		gnmi.Await(t, td.dut, sp.Static(td.staticIPv4.cidr(t)).Prefix().State(), 30*time.Second, td.staticIPv4.cidr(t))
		gnmi.Await(t, td.dut, sp.Static(td.staticIPv6.cidr(t)).Prefix().State(), 30*time.Second, td.staticIPv6.cidr(t))

File: proto/metadata.proto (hunk 2 above)
Comment (mastarkey) [reply proto/metadata.proto]: Revised comments

Comment (mastarkey) [reply proto/metadata.proto]: @nandanarista I have resolved the comments let me know if there are any issues with these comments. If so can you add more specifics on what you may be looking for. and if no further issues please approve and merge if no further changes requested

Comment (nandanarista) [reply proto/metadata.proto]: lgtm, i've approved this, but it doesn't count as an approval required for merge, you'll need a maintainer to approve.

**Comment by mastarkey** [comment]
@ram-mac 

is there any suggested time here we should adjust this to ? i can make those changes else please approve this if there are no other comments.

```
gnmi.Await(t, td.dut, sp.Static(td.staticIPv4.cidr(t)).Prefix().State(), 30*time.Second, td.staticIPv4.cidr(t))
gnmi.Await(t, td.dut, sp.Static(td.staticIPv6.cidr(t)).Prefix().State(), 30*time.Second, td.staticIPv6.cidr(t))
```

**Review by ram-mac — COMMENTED** [review]
Would suggest 120 seconds wait time where i have seen consistent results. Please add these changes to the PR and i will approve it right away.

**Comment by mastarkey** [comment]
@ram-mac i have adjusted to 120 if nothing pending please approve

**Review by ram-mac — APPROVED** [review]
LGTM

**Pull request merged**